assets/js.js

```diff
@@ -0,0 +1,210 @@
+const errorMessage = document.querySelector('.error-message');
+const searchBtn = document.querySelector('.search-btn')
+
+
+//BEGIN - GET ALL GAME
+const getAllGames = async () => {
+    try {
+        const url = `https://cs-steam-game-api.herokuapp.com/games
+        `
+        //here is how we add a dynamic value (API KEY) to the url
+        const res = await fetch(url)
+        const data = await res.json()
+        // console.log("data", data) //have a look the retrieved data
+        return data
+    } catch (error) {
+        console.log('error', error)
+    }
+
+}
+
+const renderGames = async () => {
+    document.querySelector('#nav-search').value =''
+    try {
+        errorMessage.style.display = 'flex'
+        const data = await getAllGames()
+        const gamesList = document.getElementById("content")
+        const ulGamesList = gamesList.children[1]
+        ulGamesList.innerHTML = ""
+
+        data.data.forEach((game) => {
+
+            const x = document.createElement("div")
+            x.innerHTML = `
+            <div id=${game.appid} onClick="getGamesByClick(this)" class="game-wrraper">
+                <div class="cover">
+                    <img src="${game.header_image}"
+                        alt="">
+                    <div class="game-info">
+                        <p>${game.name}</p>
+                        <p>${game.price}$</p>
+                    </div>
+                </div>
+            </div>    
+            `;
+            ulGamesList.appendChild(x)
+        })
+        errorMessage.style.display = 'none';
+    } catch (err) {
+        console.log("err", err)
+    }
+}
+//END- GET ALL GAME
+
+//BEGIN - GET SOME GENRES
+const getGenres = async () => {
+    try {
+        const url = `https://cs-steam-game-api.herokuapp.com/genres?page=2`
+        //here is how we add a dynamic value (API KEY) to the url
+        const res = await fetch(url)
+        const genres = await res.json()
+        // console.log("genres", genres) //have a look the retrieved data
+        return genres
+    } catch (error) {
+        console.log('error', error)
+    }
+
+}
+const renderGenres = async () => {
+    try {
+        const genres = await getGenres()
+        const genresList = document.getElementById("game-type")
+        let temp = 0;
+        genres.data.forEach((gen) => {
```

Comment:
have you checked this code? if I understand the code correctly, each loop you will create something like 
```
<ul>
<li><button value=${gen.name} onClick="getGamesByGenres(this);" class="btn-game-type">${gen.name}</button></li>
</ul>
```

are you sure you want that?